setup.py

```diff
@@ -26,7 +26,7 @@
 SRC_DIR = os.path.join(TOP_DIR, "onnx")
 TP_DIR = os.path.join(TOP_DIR, "third_party")
 CMAKE_BUILD_DIR = os.path.join(TOP_DIR, ".setuptools-cmake-build")
-PACKAGE_NAME = "onnx"
+PACKAGE_NAME = "onnx-protobufv21"
```

Comment:
What does this do?

Reply:
Good catch. This was for testing the built wheel with PyTorch... Here is the PR to fix this: https://github.com/onnx/onnx/pull/5089.